BrainPortal/app/controllers/nh_projects_controller.rb

```diff
@@ -58,10 +60,11 @@ def index  #:nodoc:
 
   def edit #:nodoc:
     @nh_project = find_nh_project(current_user, params[:id])
+    @can_add_license = @nh_project.creator_id == current_user.id
```

Comment:
Maybe the User with role Admin should be able to add license too.

Reply:
for now we use db or console for that